src/api/app/models/rabbitmq_bus.rb

```diff
@@ -0,0 +1,23 @@
+class RabbitmqBus
```

Comment:
[Nitpick] I would say that this should go in `lib` not in `app/models` because models is only for business logic.